Pax.sln

```diff
@@ -0,0 +1,59 @@
+﻿

```

Comment:
This was an accidental add. I will remove it from this PR - I think it would be better to add it separately without all my user-specific stuff and change/remove `build.sh` at the same time.